newIDE/app/src/SceneEditor/index.js

```diff
@@ -225,6 +226,9 @@ export default class SceneEditor extends React.Component<Props, State> {
     if (this.state.history !== prevState.history)
       if (this.props.unsavedChanges)
         this.props.unsavedChanges.triggerUnsavedChanges();
+    if (!prevProps.isActive && this.props.isActive) {
+      if (this._objectGroupsList) this._objectGroupsList.forceUpdate();
```

Comment:
I suggest we rather modify this method:
<img width="619" alt="image" src="https://user-images.githubusercontent.com/1280130/223996489-34d3dd39-803c-45d8-90e6-bb5576069055.png">


and update comment:
<img width="804" alt="image" src="https://user-images.githubusercontent.com/1280130/223996357-422d0403-b6b7-4697-a106-e4d4f90eb797.png">

so we stay consistent with how the objects list is also force updated. (or adapt the objects list so that it's force updated like you did, and clean the forceUpdateEditor if not required anymore)

Reply:
Oh good idea!

Reply:
To be honest, the way it's done for objects list is harder maybe to discover, because it's done outside the component (so it's more imperative than what you did).
But at least it will be consistent and easy to find where these force updates are done :)